src/event/event_epoll.c

```diff
@@ -194,7 +202,7 @@ _dispatch_unote_register(dispatch_unote_t du, dispatch_priority_t pri)
 	case DISPATCH_EVFILT_CUSTOM_ADD:
 	case DISPATCH_EVFILT_CUSTOM_OR:
 	case DISPATCH_EVFILT_CUSTOM_REPLACE:
-		du._du->du_wlh = DISPATCH_WLH_GLOBAL;
+		du._du->du_wlh = wlh;
```

Comment:
no, event_kevent.c also sets it to 'wlh'. I think these are arguably both wrong and should both be the GLOBAL constant (but it also doesn't really matter for this 'synthetic event' case)